Sources/App/Models/Reference.swift

```diff
@@ -3,7 +3,11 @@ import Foundation
 
 enum Reference: Equatable {
     case branch(String)
-    case tag(SemVer)
+    case tag(SemVer, _ tagName: String)
```

Comment:
Ah, interesting. Avoiding another database field. At some point, this can go too far but I think this is probably OK since they are basically the same data at this point.